stripe/src/main/java/com/stripe/android/paymentsheet/AddButton.kt

```diff
@@ -17,27 +20,84 @@ internal class AddButton @JvmOverloads constructor(
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
 ) : PrimaryButton(context, attrs, defStyleAttr) {
+    private val animator = PrimaryButtonAnimator(context)
+
     internal val viewBinding = PaymentSheetAddButtonBinding.inflate(
         LayoutInflater.from(context),
         this
     )
 
+    private val confirmedIcon = viewBinding.confirmedIcon
+
+    private val _completedAnimation = MutableLiveData<AddButtonViewState.Completed>()
+    internal val completedAnimation = _completedAnimation.distinctUntilChanged()
+
+    private var viewState: AddButtonViewState? = AddButtonViewState.Ready
```

Comment:
Do we still need this class? Can we further consolidate logic between `BuyButton` and `AddButton`?